mbf_utility/src/robot_information.cpp

```diff
@@ -65,10 +69,20 @@ bool RobotInformation::getRobotPose(geometry_msgs::PoseStamped &robot_pose) cons
   return true;
 }
 
-bool RobotInformation::getRobotVelocity(geometry_msgs::TwistStamped &robot_velocity, ros::Duration look_back_duration) const
+bool RobotInformation::getRobotVelocity(geometry_msgs::TwistStamped &robot_velocity) const
+{
+  nav_msgs::Odometry base_odom;
+  odom_helper_.getOdom(base_odom);
```

Comment:
would it make sense to check here the data? something like 
```c++
if(!base_odom.header.seq && base_odom.header.stamp == ros::Time(0)){
  ROS_WARN("No odometry message received");
  return false;
}
```

Reply:
yes, I thought about that, but... we cannot warranty that odom is properly stamped, so I didn't take the risk

but looks like we must copy odom helper (see below), so I can do a proper implementation of this check

Reply:
now implemented